packages/web/components/error/__tests__/error-boundary.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import "@testing-library/jest-dom";
+
+import * as Sentry from "@sentry/nextjs";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+import { ErrorBoundary } from "../error-boundary";
+
+jest.mock("@sentry/nextjs");
+
+describe("ErrorBoundary", () => {
+  const ChildComponent = () => {
+    throw new Error("Test error");
+  };
+
+  beforeEach(() => {
+    jest.spyOn(console, "error").mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("should render fallback UI when an error is thrown", () => {
+    const FallbackComponent = () => <div>Something went wrong</div>;
+
+    render(
+      <ErrorBoundary FallbackComponent={FallbackComponent}>
+        <ChildComponent />
+      </ErrorBoundary>
+    );
+
+    expect(screen.getByText("Something went wrong")).toBeInTheDocument();
+  });
+
+  it("should call Sentry.captureException when an error is thrown", () => {
```

Comment:
NIT BTW we're migrating away from sentry, may be good to use some sort of generic error capturing proxy to help decouple us from the observability provider. Should already have one